src/databricksbundle/DatabricksBundle.py

```diff
@@ -38,6 +40,30 @@ def __init__(self, databricks_config: str):
     def get_config_files(self):
         return ["config.yaml", "databricks/" + self.__databricks_config]
 
+    def modify_raw_config(self, raw_config: dict) -> dict:
+        project_root_filesystem_path = os.getcwd()
+        project_root_repo_path = "<not_databricks_repo>"
```

Comment:
@guderkar This is a really bad idea. Please do not set the parameter if it does not make any sense...